tomcat9/build.gradle

```diff
@@ -7,3 +7,23 @@ dependencies {
 
     testImplementation libs.tomcat9.jasper
 }
+
+def tomcat10ProjectDir = "${rootProject.projectDir}/tomcat10"
+
+// Copy the sources from ':tomcat10'.
+task generateSources(type: Copy) {
+    from("${tomcat10ProjectDir}/src/main/java")
+    into("${project.ext.genSrcDir}/main/java")
+
+    exclude '**/ArmeriaProcessor.java'
+}
+
+tasks.compileJava.dependsOn(generateSources)
+
```

Comment:
Question) Is there no need for `compileTestJava` to depend on `generateSources` (or `compileJava`)?
I realize that this is just a copy of `tomcat8`, but this seemed odd 🤔